sugoi-api-ldap-store-provider/src/main/java/fr/insee/sugoi/store/ldap/LdapWriterStore.java

```diff
@@ -910,4 +930,19 @@ public ProviderResponse deleteUserFromManagerGroup(
       return response;
     }
   }
+
+  private Collection<Attribute> getAttributesWithoutEmptyValue(Collection<Attribute> attributes) {
+    return attributes.stream()
+        .map(this::getAttributeWithoutEmptyValues)
+        .filter(attribute -> attribute.getValues().length > 0)
+        .collect(Collectors.toList());
+  }
+
+  private Attribute getAttributeWithoutEmptyValues(Attribute attribute) {
+    ASN1OctetString[] valuesFiltered =
+        Arrays.asList(attribute.getRawValues()).stream()
+            .filter(value -> value.getValueLength() > 0)
+            .toArray(ASN1OctetString[]::new);
+    return new Attribute(attribute.getName(), valuesFiltered);
+  }
```

Comment:
I would rather put this in transformSugoiToAttribute in GenericLdapMapper. Either by adding a createAttributes function like in this PR https://github.com/InseeFr/sugoi-api/pull/566/files or by having an operation type that is checked to know if attributes should be remove (it will also cover the current isToWrite boolean in mapObjectToLdapAttributes or by having a specific function in GenericLdapMapper in charge of creating all the delete modifications (this way it can also cover the removing of the last element in a list)